{"uuid": "779b3258-ddb0-40b5-8a5e-0c8c58681de0", "children": ["22985abc-f54f-4cd8-9b71-67ba2d7db1fc", "29ffe5ae-eb7c-44cf-846f-709947db5b9a", "82b1ccfe-ae6a-4c46-a332-ab66e56d0bde", "e338cf66-bf9d-475f-899e-7e390ec1df35"], "befores": [{"name": "setup", "status": "passed", "start": 1693482180250, "stop": 1693482211072}], "afters": [{"name": "setup::0", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=54011): Max retries exceeded with url: /session/5ccecf3b983b3003e4cdea1121c55dc9/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10c3c99d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n", "trace": "  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/megha/PycharmProject/pywebautomationframework/tests/vwo_login/conftest.py\", line 15, in setup\n    driver.close()\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 456, in close\n    self.execute(Command.CLOSE)\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 343, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 291, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 312, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/_request_methods.py\", line 110, in request\n    return self.request_encode_url(\n           ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/_request_methods.py\", line 143, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/poolmanager.py\", line 443, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 844, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/util/retry.py\", line 515, in increment\n    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1693482218607, "stop": 1693482218626}], "start": 1693482180250, "stop": 1693482218627}